src/UI/state/map/actions/downloadModelOutput.ts

```diff
@@ -0,0 +1,20 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { downloadModelOutputData } from '../../../api/api';
+
+export const downloadModelOutput = (blobName: string, blobLocation: string) =>
+  createAsyncThunk('map/downloadModelOutput', async () => {
+    const fileBlob = await downloadModelOutputData(blobLocation);
+
+    var link = document.createElement('a'); // once we have the file buffer BLOB from the post request we simply need to send a GET request to retrieve the file data
```

Comment:
Not sure if you want to keep this in for future reference or if you would like to remove it

Reply:
ah no, it should be removed - good spot

Reply:
Deleted now.